README.md

```diff
@@ -76,8 +140,8 @@ Let's consider some examples of routing in codec box.
 ### Split on 'publish' pins
 
 For example, you have got big source data stream, and you want to split them on some pins via session alias.
```

Comment:
For example, you **got** a big source data stream, and you want to split them **into** some pins via session alias.